src/lib/utils/number.ts

```diff
@@ -12,3 +12,17 @@ export const formatEthers = (number: string) => {
 	const asNumber = Number(ethers.utils.formatEther(number));
 	return formatNumber(asNumber);
 };
+
+export const ONE_DECIMAL_PLACE_VALUE = 1;
+export const TWO_DECIMAL_PLACE_VALUE = 2;
+export const THREE_DECIMAL_PLACE_VALUE = 3;
```

Comment:
I think we can remove these const and just pass the number unless we want to restrict the decimal places just to 3.

Reply:
sounds good to me, I'll remove these and update tests